### DISCUSSION TITLE
Adding css class to html and body tag?

### DISCUSSION THREAD

**Discussion by geetfun** [nuxt/nuxt#17521] [Questions] [answered]
Previously, I added css class names to `html` and `body` tags via app.html. For nuxt 3, [it doesn't look like app.html is supported](https://github.com/nuxt/framework/pull/2339) (? yet). 

Is there a recommended way to add css classes to `html` and `body` tags?

Thanks!

**Comment by geetfun** [comment, marked as answer]
Answering my own question... https://v3.nuxtjs.org/docs/usage/meta-tags/

From the example:

```js
export default {
  setup () {
    useMeta({
      meta: [
        { name: 'viewport', content: 'width=device-width, initial-scale=1, maximum-scale=1' }
      ],
      bodyAttrs: {
        class: 'test'
      }
    })
  }
}
```

**Reply by grantholle** [reply]
Since this answer, the API must have changed because the function is called `useHead` now... But thank you for the link (and the Nuxt team for redirecting to the new docs page)!
